dispatches/case_studies/renewables_case/prescient_options.py

```diff
@@ -15,9 +15,10 @@
 n_tracking_hour = 1
 
 solvers_list = ["gurobi_direct", "xpress_direct", "cbc"]
-opt = None
+opt = False
 for solver_name in solvers_list:
     if pyo.SolverFactory(solver_name).available(exception_flag=False):
+        opt = True
         break
 if not opt:
     raise RuntimeWarning("No available solvers")
```

Comment:
Not really relevant to this PR, but this code could use Pythons `for/else` construct in place of the boolean `opt`.